Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1

```diff
@@ -252,7 +306,7 @@ Function Invoke-RebuildSolution {
         [string] $msBuildFilePath = $env:msbuild_exe
     )
     Invoke-Task -name $MyInvocation.MyCommand.Name -task {
-        if ((Get-DeployConfig).Engine -eq 'PostgreSQL') { $buildConfiguration = 'Npgsql' }
+        if ((Get-DeploymentSettings).Engine -eq 'PostgreSQL') { $buildConfiguration = 'Npgsql' }
```

Comment:
eewww

Reply:
I'm going to create a ticket to address deployment in general